modules/apps/users-admin/users-admin-web/src/main/java/com/liferay/users/admin/web/internal/display/context/EditContactInformationDisplayContext.java

```diff
@@ -76,6 +76,8 @@ public void setPortletDisplay(
 		if (!portletName.equals(UsersAdminPortletKeys.MY_ACCOUNT)) {
 			portletDisplay.setShowBackIcon(true);
 			portletDisplay.setURLBack(getRedirect());
+			portletDisplay.setURLBackTitle(
+				portletDisplay.getPortletDisplayName());
```

Comment:
I think this is not fully correct because the previous page is not Users and Organizations (like the portletDisplayName will returns) but the Edit ***.
Probably we should instead use the same as the portletTitle